tree/master/packages/extension/public/locales/pl/translation.json

```
@@ -0,0 +1,162 @@
+	  "Accounts": "Konta",
+	  "Authorize": "Autoryzuj",
+	  "Only approve this request if you trust the application. Approving gives the application access to the addresses of your accounts.": "Zatwierdź tę prośbę tylko wtedy, gdy ufasz aplikacji. Zatwierdzenie daje aplikacji dostęp do adresów Twoich kont.",
+	  "Yes, allow this application access": "Tak, zezwól tej aplikacji na dostęp",
+	  "Add the account with the generated seed": "Dodaj konto z wygenerowanym seedem",
+	  "Create an account": "Utwórz konto",
+	  "Please write down your wallet's mnemonic seed and keep it in a safe place. The mnemonic can be used to restore your wallet. Keep it carefully to not lose your assets.": "Proszę zapisać ziarno mnemoniczne portfela i przechowywać je w bezpiecznym miejscu. Mnemonik może być użyty do przywrócenia portfela. Przechowuj go ostrożnie, aby nie stracić swoich aktywów.",
+	  "I have saved my mnemonic seed safely.": "Bezpiecznie zapisałem moje ziarno mnemoniczne.",
+	  "Next step": "Następny krok",
+	  "Derivation Path": "Ścieżka Wyprowadzenia",
+	  "Derivation Path (unlock to edit)": "Ścieżka wyprowadzania (odblokuj, aby edytować)",
+	  "//hard/soft": "//twarda/miękka ",
+	  "Add new account": "Dodaj nowe konto",
+	  "Create derived account": "Utwórz konto pochodne",
+	  "Derive new account from existing": "Wyprowadź nowe konto z istniejącego",
+	  "Choose Parent Account:": "Wybierz Konto Rodzica:",
+	  "enter the password for the account you want to derive from": "wpisz hasło do konta, z którego chcesz uzyskać dane",
+	  "Create a derived account": "Utwórz konto pochodne",
+	  "Create account from new seed": "Utwórz konto z nowego seeda",
+	  "Export account": "Eksportu konto",
+	  "You are exporting your account. Keep it safe and don't share it with anyone.": "Eksportujesz swoje konto. Trzymaj je w bezpiecznym miejscu i nie udostępniaj nikomu.",
+	  "password for this account": "hasło do tego konta",
+	  "I want to export this account": "Chcę wyeksportować to konto",
+	  "Cancel": "Anuluj",
+	  "Forget account": "Zapomnij konto",
+	  "You are about to remove the account. This means that you will not be able to access it via this extension anymore. If you wish to recover it, you would need to use the seed.": "Zamierzasz usunąć konto. Oznacza to, że nie będziesz już mógł uzyskać do niego dostępu za pośrednictwem tej wtyczki. Jeśli chcesz je odzyskać, musisz użyć seedu.",
+	  "I want to forget this account": "Chcę zapomnieć to konto",
+	  "Scan Address Qr": "Skanuj Adres Qr",
+	  "Add the account with identified address": "Dodaj konto ze zidentyfikowanym adresem",
+	  "Import account": "Importuj konto",
+	  "existing 12 or 24-word mnemonic seed": "istniejący 12 lub 24-wyrazowy ziarno mnemoniczne",
+	  "Add the account with the supplied seed": "Dodaj konto z dostarczonym materiałem seedem",
+	  "Metadata": "Metadane",
+	  "from": "od",
+	  "chain": "łańcuch",
+	  "icon": "ikona",
+	  "decimals": "dziesiętne",
```

```suggestion
	  "decimals": "miejsca dziesiętne",
```
Nie znam kontekstu, ale zakładam, że samo "dziesiętne" będzie wyglądało dziwnie.